src/index.js

```diff
@@ -10,83 +15,111 @@
         this.afters = afters || [];
         this.name = blockName;
     }
+
+    Block.factory = function (blockName, modifiers, befores, afters) {
+        return new Block(blockName, modifiers, befores, afters);
+    };
+
+    Block.clone = function (block) {
+        if (!(block instanceof Block)) {
+            return;
+        }
+
+        return new Block(
+            block.name,
+            block.modifiers.slice(),
+            block.befores.slice(),
+            block.afters.slice()
+        );
+    };
+
+    Block.prototype.el = function (el) {
+        return new Block(this.name + "__" + el);
```

Comment:
Doesn't this lose to much state?
We lose for example any modifier applied to the block.